formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I think you can remove `primary_specialist_sectors` and `additional_specialist_sectors` here, and remove a couple that are provided by `base_links.json`.